template.json

```diff
@@ -814,7 +814,7 @@
                         "base": 0,
                         "mod": []
                     },
-                    "initiation": {}
+                    "initiation": 0
```

Comment:
This causes issues with old actors having NaN limits
![image](https://github.com/SR5-FoundryVTT/SR5-FoundryVTT/assets/146255/69caa436-f0e0-4a5c-bf68-8b4050bb1392)

Reply:
I didn't think about this myself as of yet, however this will cause the limit to show up on sheet
![image](https://github.com/SR5-FoundryVTT/SR5-FoundryVTT/assets/146255/784ed21c-4ac2-4960-af41-d679b5dec4c9)

Reply:
Yeah, I forgot to change it back after testing :(

Reply:
> This causes issues with old actors having NaN limits ![image](https://user-images.githubusercontent.com/146255/283986663-69caa436-f0e0-4a5c-bf68-8b4050bb1392.png)

Thats why I wrote in the changelog a new import is required. There the field would be set. I didnt think of people using it, without doing the proposed pre-condition